include/plugin_interface/SC_Graph.h

```diff
@@ -35,6 +35,7 @@ struct Graph
 	uint32 mNumControls;
 	float *mControls;
 	float **mMapControls;
+	int32 *mAudioBusOffsets;
```

Comment:
@timblechmann changing an API from v1 to v2 gives me some pause -- will this potentially break existing code?